comps/text2image/dependency/sd_server.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""Stand-alone Stable Diffusion FastAPI Server."""
+
+import argparse
+import base64
+import os
+import time
+
+import torch
+import uvicorn
+from diffusers import DiffusionPipeline
+from fastapi import FastAPI, Request
+from fastapi.responses import JSONResponse, Response
+
+app = FastAPI()
+
+
+@app.post("/generate")
```

Comment:
why not use opea microservice decorators?

Reply:
Use opea microservice decorators would throw error on Gaudi.